tests/examples.py

```diff
@@ -1686,20 +1686,37 @@
 
 QUERY_DATA_MAPPINGS = {
     (
-        "SELECT  default_DOT_dispatcher.company_name,\n\t"
-        "avg(default_DOT_repair_order_details.price) AS default_DOT_avg_repair_price,"
-        "\n\tcount(default_DOT_repair_orders.repair_order_id) default_DOT_num_repair_orders "
-        "\n FROM roads.repair_order_details AS default_DOT_repair_order_details LEFT OUTER JOIN "
-        "(SELECT  default_DOT_repair_orders.dispatcher_id,\n\t"
-        "default_DOT_repair_orders.hard_hat_id,\n\tdefault_DOT_repair_orders.municipality_id,"
-        "\n\tdefault_DOT_repair_orders.repair_order_id \n FROM roads.repair_orders AS "
-        "default_DOT_repair_orders) AS default_DOT_repair_order ON "
-        "default_DOT_repair_order_details.repair_order_id = "
-        "default_DOT_repair_order.repair_order_id\nLEFT OUTER JOIN (SELECT  "
-        "default_DOT_dispatchers.company_name,\n\tdefault_DOT_dispatchers.dispatcher_id "
-        "\n FROM roads.dispatchers AS default_DOT_dispatchers) AS default_DOT_dispatcher "
-        "ON default_DOT_repair_order.dispatcher_id = default_DOT_dispatcher.dispatcher_id "
-        "\n GROUP BY  default_DOT_dispatcher.company_name\n LIMIT 10"
+        "WITHm0_default_DOT_num_repair_ordersAS(SELECTdefault_DOT_dispatcher.company_name,\t"
```

Comment:
Wondering why this text has no spaces in it, e.g.: "WITHm0_default_DOT_num_repair_ordersAS"